docs/news/aragon.md

```diff
@@ -1,20 +1,13 @@
 # News / Aragon
 
 ## **Featured**
-[<h2>Title</h2>](URI_to_news) |
-:-----------|
-[_Tag_](tag.md) |
-![](../../images/monthly_no_image.png) |
-_Author [Author Name](URI_to_author_profile) on Month Date#_ |
-**Subtitle** Description of the content in the news. |
-[Read More](URI_to_news) |
 
-| | |  
-:-----------:|:-----------:|:-----------:|  
-| | |  
-| | |  
-| | |  
-| | |  
-| | |  
+[<h2>Enabling crypto networks to become cross-chain using Witnet</h2>](https://medium.com/witnet/enabling-crypto-networks-to-become-cross-chain-using-witnet-2c8d3731fcb5)| 
+:-----------:|
```

Comment:
Small error here, makes the item centered, should be `:-----------|` for a featured item.